engine/src/test/scala/cromwell/webservice/routes/CromwellApiServiceSpec.scala

```diff
@@ -617,7 +617,7 @@ object CromwellApiServiceSpec {
               s"[reading back DescribeRequest contents] version: ${sourceFiles.workflowTypeVersion}"
             )
 
-            sender ! DescribeSuccess(description = WorkflowDescription(valid = true, readBack))
+            sender ! DescribeSuccess(description = WorkflowDescription.empty.copy(valid = true, errors = readBack))
```

Comment:
TOL this feels odd. Perhaps a companion object `apply` that sets the defaults instead??

Reply:
I somehow forgot about `apply` and insisted on reinventing it myself